shell/initialize/client.js

```diff
@@ -432,52 +265,6 @@ function checkForErrors(app) {
   }
 }
 
-// When navigating on a different route but the same component is used, Vue.js
-// Will not update the instance data, so we have to update $data ourselves
-function fixPrepatch(to, ___) {
-  if (this._routeChanged === false && this._paramChanged === false && this._queryChanged === false) {
-    return;
-  }
-
-  const instances = getMatchedComponentsInstances(to);
-  const Components = getMatchedComponents(to);
-
-  Vue.nextTick(() => {
-    instances.forEach((instance, i) => {
-      if (!instance || instance._isDestroyed) {
-        return;
-      }
-
-      if (
-        instance.constructor._dataRefresh &&
-        Components[i] === instance.constructor &&
-        instance.$vnode.data.keepAlive !== true &&
-        typeof instance.constructor.options.data === 'function'
-      ) {
-        const newData = instance.constructor.options.data.call(instance);
-
-        for (const key in newData) {
-          Vue.set(instance.$data, key, newData[key]);
-        }
-      }
-    });
-
-    checkForErrors(this);
-  });
-}
-
-function nuxtReady(_app) {
-  window.onNuxtReadyCbs.forEach((cb) => {
```

Comment:
`onNuxtReadyCbs` wenre't being added because there weren't any invocations of `onNuxtReady`. Without `onNuxtReadyCbs` this invocation is useless.